### ISSUE TITLE
Crash when using mysterious page

### ISSUE THREAD

**Issue by Samrel556** [MrJake222/AUNIS#137]
Minecraft Crashes in a Creative world when using a Mysterious Page. 

Crash Report: https://paste.dimdev.org/ficehutuva.mccrash
Java: Openj8 version 242
Forge: 14.23.5.2854
Modpack: Custom [modlist.txt](https://github.com/MrJake222/AUNIS/files/4495863/modlist.txt)
Aunis Version: 1.9.4

This is a testing world using Realistic Terrain Gen world gen. When I right click VanillaFix pops up saying there was an error related to Aunis. It creates the given crash report. Happens when ever a mysterious page that is created from JEI is right clicked. A page with an address is not generated.

**Comment by Samrel556** [comment]
Just tested with 1.9.5 and same everything.

**Comment by PoltixeTheDerg** [comment]
can it be replicated with the minimum required mods for AUNIS?

**Comment by PoltixeTheDerg** [comment]
if not, slowly readd mods until you can get the crash to happen, relay the results here, it will help with debugging a lot

**Comment by Samrel556** [comment]
Ok. Seems like it doesn't like [Realistic Terrain Gen](https://github.com/Team-RTG/Realistic-Terrain-Generation). It doesn't fully crash just logs that it can't update neighbors like it says in the original crash report. [latest.log](https://github.com/MrJake222/AUNIS/files/4498295/latest.log) (found on line 175)

**Comment by Samrel556** [comment]
Ok. I was messing around with the original modpack sans Realistic Terrain gen and I used the mysterious page multiple times. After the first one it quite making one with an address and threw the same error as when RTG is installed. I then jumped into the test pack with just the things for Aunis to run and got 3 mysterious pages to work before the error popped up. [latest.log](https://github.com/MrJake222/AUNIS/files/4501078/latest.log)This only has Aunis, Thermal foundation, Thermal Dynamics, Thermal Expansion, COFH core and World, CodeChickenLib, Redstone flux, and JEI
